ts/client/src/client.ts

```diff
@@ -988,7 +983,9 @@ export class MangoClient {
         bank.mint,
         U64_MAX_BN,
         false,
-        [...accountsToExclude],
+        banksToExclude,
+        perpMarketsToExclude,
+        serum3MarketsToExclude,
```

Comment:
mmh, this probably works and I'm ok with it. However, in other places we adjust (a copy of) the mangoAccount to make the health account generation do the right thing. Might it have been easier to deactivate the perp, serum, token accounts in mangoAccount after each ix is made? Then those wouldn't need to be passed explicitly in the exclusion lists.